app.js

```diff
@@ -4,6 +4,63 @@ const close = document.querySelector('.close-icon');
 const myEmail = document.querySelector('.e-mail');
 const emailIcon = document.querySelector('.e-mail-icon-style');
 const liPortfolio = document.querySelector('.mg-t-dom');
+const backdrop = document.createElement('section');
+const modal = document.createElement('div');
+const startSection = document.querySelector('.hr1');
+const card = {
+  images: ['img/img-placeholder.svg', 'img/img-placeholder.svg', 'img/img-placeholder.svg', 'img/img-placeholder.svg'],
+  name: ['Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories', 'Multi-Post Stories'],
+  description: 'A daily selection of privately personalized reads no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+  technologies: ['css', 'html', 'Bootstrap', 'Ruby'],
+  liveButton: '',
+  sourceButton: '',
+  image: 'img/modal-photo.svg',
+};
+
+function closeModal() {
+  if (backdrop) {
+    backdrop.remove();
+  }
+  if (modal) {
+    modal.remove();
+  }
+}
+
+function popupWindow(image, workTitle) {
+  modal.classList.add('modal');
+  modal.innerHTML = `
+    <div class="popup-title">
+      <h5 class="g-item2 pff-500 inner-title">${workTitle}</h5>
+      <img class="close-popup inner-close-icon" src="img/close-icon.svg" alt="Close popup">
+    </div>
+    <div class="image-popup">
+      <img class="images" src="${image}" alt="Project Image">
+    </div>
+    <div class="popup-pharagraph">
+          <p class="g-item3 paragraph-popup">Lorem Ipsum is simply dummy text 
+          of the printing and typesetting industry. Lorem Ipsum has been the 
+          industry's standard dummy text ever since the 1500s, when an unknown 
+          printer took a galley of type and scrambled it 1960s with the 
+          releaLorem Ipsum is simply dummy text of the printing and typesetting  
+          ever since the 1500s, when an unknown printer took a galley of type 
+          veris lapoa todoe.</p>
+    </div>
+    <div class="popup-tag">
+      <ul class="container ul-card">
+        <li class="li-card g-item4"><a class="sff-500 a-card">${card.technologies[0]}</a></li>
+        <li class="li-card g-item5"><a class="sff-500 a-card">Ruby on Rails</a></li>
+        <li class="li-card g-item5"><a class="sff-500 a-card">${card.technologies[1]}</a></li>
+      </ul>
+    </div>
+    <div class="c-container popup-buttons">
+        <button class="g-item8 pff-500 inner-margin-buttom" type="button"><a class="sff-500 a-card" href="${card.liveButton}">See Live<img class="inner-icon" src="img/live-icon.svg"></a></button>
+        <button class="g-item8 pff-500 inner-margin-buttom" type="button"><a class="sff-500 a-card" href="${card.sourceButton}">See Source<img class="inner-icon" src="img/github-icon.svg"></a></button>
+    </div>`;
+  backdrop.appendChild(modal);
+  const closePopup = document.querySelector('.close-popup');
+  closePopup.addEventListener('click', closeModal);
+}
+
```

Comment:
Kindly try to match the Figma design in the desktop version :
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156564260-767c8fb3-3b61-4865-9ec8-3c30fa7b8f8e.png) | ![image](https://user-images.githubusercontent.com/24703171/156564339-91fdd457-169c-4c01-be87-04b9b757fea5.png) |

- [ ] Please increase the left and right spacing of the paragraph.
- [ ] Kindly try to center the technologies list to match the Figma design.
- [ ] Kindly make the `see live` and `see source` beside each other and increase its width to match the Figma design.

Reply:
resolved